src/current/v23.1/cdc-queries.md

```diff
@@ -174,18 +174,22 @@ To emit the previous state of a row, it is necessary to explicitly call `cdc_pre
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-CREATE CHANGEFEED INTO 'external://sink' AS SELECT *, cdc_prev FROM movr.rides;
+CREATE CHANGEFEED INTO 'external://sink' AS SELECT rider_id, vehicle_id, cdc_prev FROM movr.rides;
 ~~~
 
 To emit the previous state of a column, you can specify this as a named field from the `cdc_prev` tuple with the following syntax:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-CREATE CHANGEFEED INTO 'external://sink' AS SELECT *, cdc_prev FROM movr.vehicles WHERE (cdc_prev).status = 'in_use';
+CREATE CHANGEFEED INTO 'external://sink' AS SELECT owner_id, current_location, cdc_prev FROM movr.vehicles WHERE (cdc_prev).status = 'in_use';
```

Comment:
perhaps SELECT (cdc_prev).owner_id as old_owner_id, (cdc_prev).status as old_status to make it clear that you don't need to select the entire "cdc_prev" and can also limit it to the columns that are interesting.

It's okay if you think that omitting this makes it clearer.

Reply:
I have updated — it seems a better fit here for the example to only request the `cdc_prev` data on a column.